examples/kafka/src/main/scala/org/apache/gearpump/streaming/examples/kafka/wordcount/KafkaWordCount.scala

```diff
@@ -59,6 +62,9 @@ object KafkaWordCount extends App with ArgumentsParser {
 
   val config = parse(args)
   val context = ClientContext(config.getString("master"))
+
+  implicit val system = context.system
```

Comment:
ok, I see. Is there a way to hide this ? Like for the KafkaStreamProcessor, how about inheriting it from TaskActor ?